docs/config/_default/config.toml

```diff
@@ -50,7 +50,7 @@ writeStats = true
 
 ############################# Outputs ##############################
 [outputs]
-home = ["HTML"]
+ home = ["HTML", "JSON"]

```

Comment:
Adding `JSON` as an output allows us to build JSON files directly via Hugo instead of adding a library.